src/robotide/lib/robot/writer/formatters.py

```diff
@@ -61,7 +61,7 @@ def _is_indented_table(self, table):
 
     def _escape_consecutive_whitespace(self, row):
         return [self._whitespace.sub(self._whitespace_escaper,
-                                     cell.replace('\n', ' ')) for cell in row]
+                                     cell.replace('\n', '${\\n}')) for cell in row]
```

Comment:
Why not just replace '\n' with '\\n'?

Reply:
This is not a required change, or what would be our benefit from it?
Usually we don't change the files under ../lib/robot they are a copy of RobotFramework, in this case v3.1.2.

On my fork, on branch `rf32` I am upgrading that library to v3.2.1. Which is really problematic. I created a `compat` directory to help making a transition between the old and new parser. I am doing a break, I have a minimal convertion working, it runs without errors but the object in the Tree are not OK, yet. I'll see if I can rebase to current development, and  I hope I can get your collaboration (@Nyral maybe we can talk about this). Anyway, this is not planned for v2.0, because it is taking too long.

Reply:
> Usually we don't change the files under ../lib/robot they are a copy of RobotFramework, in this case v3.1.2.
I think this improvement could be made without modifying any of the files from there.

> what would be our benefit from it?
I can see some benefit in making tests look better. Say for example that I want to log some data. This change will make it look exactly like in report.
![image](https://user-images.githubusercontent.com/39315668/88453770-d8603a00-ce72-11ea-98ac-10b1f8644378.png)

Reply:
> Why not just replace \n with \\n?

Actually I tried, did not work somehow.

>> Usually we don't change the files under ../lib/robot they are a copy of RobotFramework, in this case v3.1.2.

> I think this improvement could be made without modifying any of the files from there. Good point.

You're correct, let me find some better solution.

Reply:
We would like to have visualization/navigation with hidden \n, but keep them on edit cell mode.